lib/src/components/NxToast/NxToast.scss

```diff
@@ -5,28 +5,21 @@
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
 
-.nx-toast-wrapper {
-  // nx-toast-wrapper has position absolute to ensure toasts stay in the top right corner of the page.
-  // Using display:flex and flex-end ensures the right side of the toast is aligned with the right side of 
-  // nx-page-main. The nx-toast-container has position fixed applied so it remains fixed while scrolling.
-  position: absolute;
-  top: 0;
-  right: 0;
-  display: flex;
-  justify-content: flex-end;
-  margin-right: var(--nx-spacing-6x);
-}
-
 .nx-toast-container {
-  position: fixed;
+  position: sticky;
```

Comment:
Why sticky?

Reply:
> Why sticky?

I found that with position: fixed, it was positioning itself at the top right corner and in front of the header if it was present. Sticky was a bit cleaner that it seems to position itself at the top right corner of the screen without a header, and underneath the header if one is present.

Reply:
> I found that with position: fixed, it was positioning itself at the top right corner and in front of the header if it was present. Sticky was a bit cleaner that it seems to position itself at the top right corner of the screen without a header, and underneath the header if one is present.

What about `absolute` though? I don't think you're using the special features of `sticky`

Reply:
> What about absolute though? I don't think you're using the special features of sticky

Actually, since this `.nx-toast-container` is a grid child I think what we really want is `position: static` (the default) along with `justify-self: end`

Reply:
> Actually, since this `.nx-toast-container` is a grid child I think what we really want is `position: static` (the default) along with `justify-self: end`

Ah, I hadn't thought about `justify-self: end`. Worked like a charm with position static!